chapter_preliminaries/probability.md

```diff
@@ -354,54 +808,235 @@ $$P(H = 1| D_1 = 1, D_2 = 1)
-The astute reader might have noticed that the diagnosis behaved like a classifier hiding in plain sight where our ability to decide whether a patient is healthy increases as we obtain more features (test outcomes). We will pick up this idea in :ref:`sec_naive_bayes` where we will inroduce Naive Bayes Classifiers, using the approximation that all features occur independently. 
+That is, the second test allowed us to gain much higher confidence that not all is well.
+Despite the second test being considerably less accurate than the first one, 
+it still significantly improved our estimate. 
+The assumption of both tests being conditional independent of each other 
+was crucial for our ability to generate a more accurate estimate.
+Take the extreme case where we run the same test twice. 
+In this situation we would expect the same outcome in both times, 
+hence no additional insight is gained from running the same test again. 
+The astute reader might have noticed that the diagnosis behaved 
+like a classifier hiding in plain sight 
+where our ability to decide whether a patient is healthy 
+increases as we obtain more features (test outcomes).
 
-## Expectations
 
-Quite often probabilities *per se* are insufficient to provide us with the relevant insight to make decisions. For instance, when we want to decide whether to make an investment we should assess the expected return and the risk profile. For instance, with 50% probability the investment might fail, with 40% probability it might return twice the investment and with 10% it might return 10 times the money invested. To calculate the expected return we sum over all returns multiplied by the probability that they will occur. This yields $0.5 \cdot 0 + 0.4 \cdot 2 + 0.1 \cdot 10 = 1.8$. Hence the expected return is 1.8 times the initial investment. 
+## Expectations
 
-In general, the *expectation* (or average) of the random variable $X$ is defined as
+Often, making decisions requires not just looking 
+at the probabilities assigned to individual events
+but composing them together into useful aggregates
+that can provide us with guidance.
+For example, when random variables take continuous scalar values,
+we often care about knowing what value to expect *on average*.
+This quantity is formally called an *expectation*.
+If we are making investments,
+the first quantity of interest
+might be the return we can expect, 
+averaging over all the possible outcomes
+(and weighting by the appropriate probabilities).
+For instance, say that with 50% probability, 
+an investment might fail altogether,
+with 40% probability it might provide a 2x return,
+and with 10% probability it might provide a 10x return 10x.
+To calculate the expected return,
+we sum over all returns, multiplying each
+by the probability that they will occur. 
+This yields the expectation 
+$0.5 \cdot 0 + 0.4 \cdot 2 + 0.1 \cdot 10 = 1.8$. 
+Hence the expected return is 80%
+(1.8 times the initial investment).
```

Comment:
Since we used "2$\times$/10$\times$ returns", simply saying "the expected return is 1.8$\times$" is less confusing.